starter-code/css/main.css

```diff
@@ -6,3 +6,92 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+font-family: Circular, Arial, Helvetica, sans-serif;
+
+}
+
+.navbar{
+background-color:white;
+position : fixed;
```

Comment:
You don't have to fix it because it will tend to block the position even when you scroll down.